src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -410,6 +410,7 @@ export const enum TERMINAL_COMMAND_ID {
 	NEW_WITH_CWD = 'workbench.action.terminal.newWithCwd',
 	NEW_LOCAL = 'workbench.action.terminal.newLocal',
 	NEW_IN_ACTIVE_WORKSPACE = 'workbench.action.terminal.newInActiveWorkspace',
+	NEW_FROM_PROFILE = 'workbench.action.terminal.newFromProfile',
 	NEW_WITH_PROFILE = 'workbench.action.terminal.newWithProfile',
```

Comment:
These names are awfully similar, not clear at all what the difference is

Reply:
thanks, forgot to delete - `NEW_FROM_PROFILE` is no longer used since I consolidated the methods